translations/ui-orders/en.json

```diff
@@ -6,15 +6,6 @@
   "deletePiece.delete.success": "Selected pieces were deleted",
   "deletePiece.delete.fail": "Selected pieces were not deleted",
   "deletePiece.delete.thereAreRequestsOnItem": "Selected pieces were not deleted. There are requests on item",
-  "acquisition_method.approvalPlan": "Approval plan",
```

Comment:
hm, not sure that default should be always in English, please check how it's done with close reasons

mb point for Dennis

Reply:
Yes, the reasons are using translations, I'll make some updates, thanks

Reply:
I updated `getAcqMethodsOptions`, now it returns options with translated default acq methods